host-bin/edit-chroot

```diff
@@ -594,6 +610,31 @@ for NAME in "$@"; do
             mv "$CHROOT" "$target"
         fi
     fi
+
+    # Copy the chroot if requested
```

Comment:
I actually built it so you can't copy and move at the same time, and included testing for this. I can't see the use case of wanting to do both, and it seems it would be more likely to be a mistake. Do you feel this needs to be reworked so that you can move while copying?